tests/test_agents.py

```diff
@@ -38,8 +38,8 @@ async def test_get_directory_index(agent_test_settings: Settings) -> None:
         "title",
         "year",
     ], "Incorrect fields in index"
-    # paper.pdf + flag_day.html + bates.txt + obama.txt
-    assert len(await index.index_files) == 4, "Incorrect number of index files"
+    # paper.pdf + empty.txt + flag_day.html + bates.txt + obama.txt
+    assert len(await index.index_files) == 5, "Incorrect number of index files"
```

Comment:
Shouldn't the empty not result in an index file?

Reply:
I think it does but tracks it as importable for future reference.

Reply:
interesting 👍

Reply:
Alright @whitead I adjusted the test to account for the return of `index_files`, which is a `dict` with a value of `"ERROR"` for `empty.txt`. So now hopefully that's more intuitive